app/src/main/java/fr/neamar/kiss/MainActivity.java

```diff
@@ -662,14 +702,13 @@ public void updateSearchRecords() {
     private void updateSearchRecords(String query) {
         resetTask();
 
-        if (mPopup != null)
-            mPopup.dismiss();
-
         forwarderManager.updateSearchRecords(query);
 
         if (query.isEmpty()) {
+            toggleTags.hideBar();
```

Comment:
Not sure if this is related to this line, but there is a bug in the current tag bar, if you search for something then empty the search query, bar is not displayed again.